docs/en/ufs/Ozone.md

```diff
@@ -84,6 +84,65 @@ $ ./bin/alluxio fs mount \
   /ozone o3fs://<OZONE_BUCKET>.<OZONE_VOLUME>/
 ```
 
+Possible core-site.xml and ozone-site.xml
+- core-site.xml
+
+```xml
+<configuration>
+  <property>
+    <name>fs.o3fs.impl</name>
+    <value>org.apache.hadoop.fs.ozone.BasicOzoneFileSystem</value>
+  </property>
+  <property>
+    <name>fs.AbstractFileSystem.o3fs.impl</name>
+    <value>org.apache.hadoop.fs.ozone.BasicOzFs</value>
+  </property>
+  <property>
+    <name>fs.defaultFS</name>
+    <value>o3fs://mybucket.myvol</value>
+  </property>
```

Comment:
I would prefer that for these configs we use the bare-minimum amount of properties. In this case, I think `fs.defaultFs` is unnecessary - the other two are ok though

Reply:
Done.